lib/modules/login/login_interactor.dart

```diff
@@ -1,7 +1,24 @@
 part of 'login_module.dart';
 
-abstract class LoginInteractor extends Interactor<LoginInteractorDelegate> {}
+abstract class LoginInteractor extends Interactor<LoginInteractorDelegate> {
+  void login(String email, String password);
+}
```

Comment:
imo, you can remove this `abstract class` and come straight to the point, making just a real Interactor class. This saves you a few lines of abstraction code. In the end, when you test this in the other place, you can always Mock it, since it doesn't contain any concrete class or depending on ANY other component. It's simple to Mock directly.

Reply:
ok @sleepylee, but let me check how this change will affect to write test first 👍

Reply:
sure